docs/releasing/publishing.md

```diff
@@ -1,17 +1,33 @@
-# Publishing GOV.UK Frontend
+# Steps you must complete before you release a new version of GOV.UK Frontend
 
-1. Make sure you have cards in the "Backlog" column to:
-  - Update the GOV.UK Design System to use the latest release
-  - Update the GOV.UK Prototype Kit to use the latest release
-  - Update the GOV.UK Frontend Docs to use the latest release
+1. Raise new issues in the team GitHub repositories ([govuk-frontend](https://github.com/alphagov/govuk-frontend), [govuk-prototype-kit](https://github.com/alphagov/govuk-prototype-kit)) to:
```

Comment:
Should this list include the Frontend Docs repo too? https://github.com/alphagov/govuk-frontend-docs

Reply:
```suggestion
1. Raise new issues in the team GitHub repositories ([govuk-frontend](https://github.com/alphagov/govuk-frontend), [govuk-frontend-docs](https://github.com/alphagov/govuk-frontend-docs), [govuk-prototype-kit](https://github.com/alphagov/govuk-prototype-kit)) to:
```